### PR TITLE
costa: support beta version for installing plugin

### PR DESCRIPTION
**Pull request by yorkie**
This supports the beta and specific version for plugin installation:

- `@pipcook/my-plugin`
- `@pipcook/my-plugin@latest`
- `@pipcook/my-plugin@beta`
- `@pipcook/my-plugin@1.0.0`

### CODE REVIEW

**Comment by yorkie** [comment]
The costa unit tests pass at https://github.com/alibaba/pipcook/pull/202/checks?check_run_id=729962421#step:5:922 :)

File: packages/costa/src/runtime.ts [hunk 1]
```diff
@@ -18,7 +19,21 @@ import Debug from 'debug';
 const debug = Debug('costa.runtime');
 const CONDA_CONFIG = path.join(__dirname, '../node_modules/@pipcook/boa/.CONDA_INSTALL_DIR');
 
-function selectLatestPackage(metadata: NpmPackageMetadata): NpmPackage {
+function selectNpmPackage(metadata: NpmPackageMetadata, source: PluginSource): NpmPackage {
+  const { version } = source?.schema;
+  if (version === 'beta') {
+    if (metadata['dist-tags'].beta == null) {
```
Comment (utkobe) [inline packages/costa/src/runtime.ts:25]: === or just !metadata['dist-tags'].beta ?

Comment (yorkie) [reply packages/costa/src/runtime.ts:25]: `==` with null accept the value `undefined` and `null`.

**Comment by utkobe** [comment]
LGTM~

**Pull request merged**